README.md

```diff
@@ -2,9 +2,11 @@
 
 This repository contains Postman collections for the commercetools platform.
 
+[Get Started here](GettingStarted.md) with commercetools postman collection.
+
 ## Disclaimer
 
-This is not the official commercetools platform documentation. Please see [here](http://docs.commercetools.com/)
+This is not the official commercetools platform documentation. To read complete documentation for the commercetools platform, please visit http://docs.commercetools.com/
 for a complete and approved documentation of the commercetools platform API.
```

Comment:
```suggestion

```

Reply:
I didn't see this extra line before. It should be removed if the preceding line has changed to my suggestion.